### ISSUE TITLE
actions 00-pr-scanner failed for external pr because I cannot set username and password for quay

### ISSUE THREAD

**Issue by MMMMMMorty** [kubescape/kubescape#1482]
# Description
I upload my pr, and it started running the actions 0-pr-scanner, but it failed during the `binary-build `, because I didn't set `QUAYIO_REGISTRY_USERNAME` and `QUAYIO_REGISTRY_PASSWORD` is set in GitHub secrets. But as an external pr, I cannot set them in GitHub secrets, and I don't have the quay account.

# Failed Log
[Failure actions](https://github.com/kubescape/kubescape/actions/runs/6941987820)
```
Login to Quary
Run docker/login-action@v2
Error: Username and password required
```

# Expected behavior
For external pr, no need to set `QUAYIO_REGISTRY_USERNAME` and `QUAYIO_REGISTRY_PASSWORD` in GitHub secrets.

**Comment by matthyx** [comment]
yes... if you have time you can add a conditional to skip this test on forks, or something like that, pls?

**Comment by MMMMMMorty** [comment]
Okay, I will take a look.

**Comment by VaibhavMalik4187** [comment]
@MMMMMMorty are you working on this issue?

**Comment by MMMMMMorty** [comment]
@VaibhavMalik4187 Thank you for your solution, but it doesn't work properly, so I created a new PR to revert [this PR](https://github.com/kubescape/kubescape/pull/1490) and fix the underlying problem.
